apps/condo/domains/common/components/UseDeskWidget.tsx

```diff
@@ -1,7 +1,7 @@
 import { useOrganization } from '@condo/next/organization'
 import React, { useEffect } from 'react'
 import getConfig from 'next/config'
-import get from 'lodash/get'
+import { get, isFunction } from 'lodash'
```

Comment:
it is better to import only to the required function. 
import get from 'lodash/get'
import get from 'lodash/isFunction'